content/md/first-steps/manage-your-attributes.md

```diff
@@ -268,31 +268,97 @@ Unlike products you cannot restore or revert an attribute’s version.
 
 # Delete an attribute
 
+There are two ways to remove an attribute:
+* Through the attribute edit form
+* Through the delete shortcut in the grid
+
+You can also choose if you want to loose or to keep the data used by this attribute.
+
+::: warning
+If you cannot access the attribute page or if you cannot add/delete an attribute, it is probably because you do not have the permission to.  
+Please refer to [Manage your user roles](build-your-user-roles.html) to learn about roles.
+:::
+
+::: warning
+Please first delete your attribute from the families it is used in and then from the products. After this, you will be able to delete safely your attribute.
+:::
+
+::: ee
+An attribute that is used in a published product cannot be deleted. You first need to unpublish the product(s) and then delete the attribute.
+:::
+
+##Delete an attribute without keeping the related data
+
 ::: warning
 Please first delete your attribute from the families it is used in and then from the products. After this, you will be able to delete safely your attribute.
 :::
 
-You have two ways to remove an attribute:
+###Through the attribute edit form
 
-Through the attribute edit form:
-1. Select the attribute to delete
-1. Click on `...` for other actions and then `Delete` on the top right corner
+1.  Go to `Settings` / `Attributes`
+1.  Select the attribute to delete
+1.  Click on `...` for other actions and then `Delete` on the top right corner
   ![image](../img/Settings_Attributes_Delete.png)
-1. Confirm the action in the popin
+1.  Confirm the action in the popin
```

Comment:
I think it's not intentional, but you added unnecessary spaces here.

Reply:
Corrected!